core/scheduler-plugin-biserver-schedule-jmeter.jmx

```diff
@@ -0,0 +1,2102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="2.8" jmeter="2.13 r1665067">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="BI Server Web Service API Test (extensions)" enabled="true">
```

Comment:
ported this jmeter file from the legacy scheduler folder. Updated the REST urls

see command how to launch jmeter: https://hv-eng.atlassian.net/browse/BACKLOG-38317?focusedCommentId=1921761